Source/Magick.NET/Shared/Profiles/Exif/ExifDefinition.cs

```diff
@@ -0,0 +1,364 @@
+﻿using System.Collections;
+
+namespace ImageMagick
+{
+    /// <summary>
+    /// Class that can be used to provide a definition of ExifValues within an EXIF profile.
+    /// </summary>
+    public class ExifDefinition
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ExifDefinition"/> class.
+        /// </summary>
+        /// <param name="exifTag"></param>
+        /// <param name="exifDataType"></param>
+        /// <param name="isArray"></param>
+        internal ExifDefinition(ExifTag exifTag, ExifDataType exifDataType, bool isArray)
+        {
+            ExifTag = exifTag;
+            ExifDataType = exifDataType;
+            IsArray = isArray;
+        }
+
+        /// <summary>
+        /// Gets a value indicating ExifTag for the current definition.
+        /// </summary>
+        public ExifTag ExifTag
+        {
+            get; 
+            private set;
+        }
+
+        /// <summary>
+        /// Gets a value indicating ExifDataType for the current definition.
+        /// </summary>
+        public ExifDataType ExifDataType 
+        {
+            get; 
+            private set;
+        }
+
+        /// <summary>
+        /// Gets a value indicating whether current definition contains an array value.
+        /// </summary>
+        public bool IsArray 
+        {
+            get; 
+            private set;
+        }
+
+        public static Hashtable ExifDefinitions { get; } = new Hashtable
```

Comment:
I rather have this in a separate class.

Reply:
OK, can you suggest a class? Is there something similar that you already do that I can mimic?